src/steps/email-links-validation.ts

```diff
@@ -55,14 +56,24 @@ export class EmailLinksValidationStep extends BaseStep implements StepInterface
         ]);
       }
 
+      const storageUrl: string = inbox.items.reverse()[position - 1].storage.url;
+      let messageRecords;
+
       if (!inbox.items[position - 1]) {
         return this.error("Email #%d hasn't been received yet: there are %d message(s) in the inbox.", [
           position,
           inbox.items.length,
         ]);
       }
 
-      const storageUrl: string = inbox.items.reverse()[position - 1].storage.url;
+      if (inbox.items.length > 1) {
```

Comment:
- [ ] Same comment as previous: `>= 1`
- [ ] Same comment as previous: return the binary record too.